music21/languageExcerpts/instrumentLookup.py

```diff
@@ -15,169 +16,141 @@
 # noinspection SpellCheckingInspection
 abbreviationToBestName = {
     'a sax': 'alto saxophone',
-    'a.': 'alto',
     'ac b': 'acoustic bass',
     'ac gtr': 'acoustic guitar',
     'acc': 'accordion',
-    'acc.': 'accordion',
     'accdn': 'accordion',
-    'arp.': 'harp',
+    'arp': 'harp',
     'b dr': 'bass drum',
-    'b.': 'bass',
```

Comment:
is there a reason this wasn't replaced by `'b': 'bass'` -- in your view, is it too ambiguous with `baritone`?

Reply:
Cleaning up this process exposed some more major issues with the method for getting pitch name from string. This is as far as I've got with unpicking it:
https://github.com/cuthbertLab/music21/pull/631/commits/4323fa22980e810dc6c0855e06b109dd42e74c38
Basically the previous 'b.' abbreviation was never called and 'b' (as well as 'a') don't work alongside the handling of transposition. So this PR is an improvement (not a trade-off) but only an incremental one for now. A more substantial fix in needed to move this further along.